Buildings/Controls/OBC/Facade/ShadingEnable.mo

```diff
@@ -0,0 +1,121 @@
+within Buildings.Controls.OBC.Facade;
+block ShadingEnable "Generic shading device enable/disable seqence"
+
+  parameter Boolean use_solIrr = false
+    "Set to true if enabling based on solar irradiance, otherwise enabling based on temeprature"
+    annotation(Dialog(group = "Conditional"));
+  parameter Modelica.SIunits.Temperature TThr = 298.15
```

Comment:
`TSet` would be clearer than `TThr` as it is a set point.

Reply:
@mwetter Setpoint is something a controlled device tracks, whereas in this case it seems to be a pure activation/deactivation threshold, especially if it is the outdoor air temperature. E+ suggests there exist setpoint schedules, which are then followed by the angle of the blind.

Reply:
I think it is still fine to call it set point. If it is the room temperature, then there could be situations where opening and closing the shade will keep the room temperature around this temperature (but typically, the control gain is too small)